src/components/modules/admin/pages/AdminMonthlyReviewsPage.tsx

```
@@ -0,0 +1,75 @@
+                        Delete
+                      </Button>
+                    </form>
+                  </div>
+                </td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+        {items.length === 0 ? (
+          <p className="px-4 py-8 text-center text-sm text-muted-foreground">
+            No monthly reviews found. Click 'Create Review' to get started!
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)